models/job_helpers_test.go

```diff
@@ -0,0 +1,255 @@
+					ArtifactID: jobs[0].Data.MongoDBBackup.ArtifactID,
+				},
+				Expect: []string{jobs[0].ID},
+			},
+		}
+
+		for _, tc := range testCases {
+			jobs, err := models.FindJobs(findTX.Querier, tc.Filters)
+			assert.NoError(t, err)
+			ids := make([]string, len(jobs))
+			for i := range jobs {
+				ids[i] = jobs[i].ID
+			}
+			sort.Strings(tc.Expect)
+			sort.Strings(ids)
+			assert.Equal(t, tc.Expect, ids)
+		}
+	})
+
+}
+func TestJobLogs(t *testing.T) {
+	sqlDB := testdb.Open(t, models.SkipFixtures, nil)
+	db := reform.NewDB(sqlDB, postgresql.Dialect, reform.NewPrintfLogger(t.Logf))
+	tx, err := db.Begin()
+	require.NoError(t, err)
+	t.Cleanup(func() {
+		require.NoError(t, tx.Rollback())
+		require.NoError(t, sqlDB.Close())
+	})
+
+	job1, err := models.CreateJob(tx.Querier, models.CreateJobParams{
+		PMMAgentID: "pmmagent",
+		Type:       models.MongoDBBackupJob,
+		Data:       &models.JobData{},
+	})
+	require.NoError(t, err)
+
+	job2, err := models.CreateJob(tx.Querier, models.CreateJobParams{
+		PMMAgentID: "pmmagent",
+		Type:       models.MongoDBBackupJob,
+		Data:       &models.JobData{},
+	})
+	require.NoError(t, err)
+
+	createRequests := []models.CreateJobLogParams{
+		{
+			JobID:   job1.ID,
+			ChunkID: 0,
+			Message: "some log",
+		},
+		{
+			JobID:   job1.ID,
+			ChunkID: 1,
+			Message: "another log",
+		},
+		{
+			JobID:   job2.ID,
+			ChunkID: 0,
+			Message: "some log",
+		},
+	}
+
+	t.Run("create", func(t *testing.T) {
+		for _, req := range createRequests {
+			log, err := models.CreateJobLog(tx.Querier, req)
+			assert.NoError(t, err)
+			assert.Equal(t, req.JobID, log.JobID)
+			assert.Equal(t, req.ChunkID, log.ChunkID)
+			assert.Equal(t, req.Message, log.Message)
+			assert.False(t, log.LastChunk)
+		}
+	})
+
+	t.Run("find", func(t *testing.T) {
+		type expectLog struct {
+			JobID   string
+			ChunkID int
+		}
+		type testCase struct {
+			Filters models.JobLogsFilter
+			Expect  []expectLog
+		}
+		testCases := []testCase{
+			{
+				Filters: models.JobLogsFilter{
+					JobID: job1.ID,
+				},
+				Expect: []expectLog{
+					{
+						JobID:   job1.ID,
+						ChunkID: 0,
+					},
+					{
+						JobID:   job1.ID,
+						ChunkID: 1,
+					},
+				},
+			},
+			{
+				Filters: models.JobLogsFilter{
+					JobID: job1.ID,
+					Limit: pointer.ToInt(1),
+				},
+				Expect: []expectLog{
+					{
+						JobID:   job1.ID,
+						ChunkID: 0,
+					},
+				},
+			},
+			{
+				Filters: models.JobLogsFilter{
+					JobID:  job1.ID,
+					Offset: 1,
+					Limit:  pointer.ToInt(1),
+				},
+				Expect: []expectLog{
+					{
+						JobID:   job1.ID,
+						ChunkID: 1,
+					},
+				},
+			},
+		}
+
+		for _, tc := range testCases {
+			logs, err := models.FindJobLogs(tx.Querier, tc.Filters)
+			assert.NoError(t, err)
+			require.Len(t, logs, len(tc.Expect))
+			for i := range logs {
+				assert.Equal(t, tc.Expect[i].JobID, logs[i].JobID)
+				assert.Equal(t, tc.Expect[i].ChunkID, logs[i].ChunkID)
+			}
+		}
```

Comment:
Let's use existing style for table tests https://github.com/percona/pmm-managed/blob/9b531e463f595d94c01dd2e45d3b6345b7c17f9c/services/management/ia/alerts_service_test.go#L111-L124